src/main/java/org/openmrs/uitestframework/test/TestData.java

```diff
@@ -599,7 +599,7 @@ public static String createLocation(String locationName){
 	
 	public static void permanetDelete(String uuid){
 		
-		if(null != uuid && !uuid.isEmpty()){
+		if(null != uuid && StringUtils.isNotBlank(uuid)){

```

Comment:
oh sorry forgot that isNotBlank checks also for "null". Now it is fixed. I guess that I have to squash my commits into one, right? if so from the branch RA-1295, I execute the command git rebase -i master, according to the guide 
https://wiki.openmrs.org/display/docs/Using+Git#UsingGit-Getthecodeinready-onlymode%28withoutintentionofcontributing%29
right?